### ISSUE TITLE
Text with mixed english and Hebrew is strangely formatted in RTL.

### ISSUE THREAD

**Issue by chrisgarrity** [LLK/scratch-blocks#1683]
Text with placeholders is by definition a mixture of RTL and LTR characters as the placeholder must be '%1' etc.
The translation for glide is:
![screen shot 2018-08-16 at 5 36 44 pm](https://user-images.githubusercontent.com/399209/44236477-221a5300-a17b-11e8-90af-a4c292424de2.png)
`גלוש %1 שניות ל x: %2 y: %3`

But this appears as:
![screen shot 2018-08-16 at 5 37 45 pm](https://user-images.githubusercontent.com/399209/44236502-365e5000-a17b-11e8-8c99-efd4379aa7b4.png)

Clearly the 'x:' is in the wrong place.

A note from a translator suggests adding this style could help as it would override all the text.
```css
text.blocklyText { direction: rtl; }
```

**Comment by guyhed** [comment]
As Chris wrote, the Hebrew version is not usable as it is now. Some blocks are unreadable, while others, like operators, work wrong. I give examples below.
It seems that the style command
`.blocklyText { direction: rtl; }`
does fix it all (at least, all the mistakes I have noticed).

For example, at present  "slide to" block looks like 
![screenshot from 2018-08-25 14-47-19](https://user-images.githubusercontent.com/1233716/44617979-30045e00-a876-11e8-9cd0-faf6765b88b6.png)
and after the implementation of the style above it looks as it should: 
![screenshot from 2018-08-25 14-46-30](https://user-images.githubusercontent.com/1233716/44617987-52967700-a876-11e8-93f5-2b4094a7f643.png)
The above style also fixes the order of the elements in the operator blocks. For example:
at present in the Hebrew version, due to inverse order of the fields, you get: 
![screenshot from 2018-08-25 14-53-05](https://user-images.githubusercontent.com/1233716/44618027-01d34e00-a877-11e8-8ab7-a15ec81af100.png)
after implementing the above style: 
![screenshot from 2018-08-25 14-53-59](https://user-images.githubusercontent.com/1233716/44618034-321aec80-a877-11e8-8b62-0dbd95b25e73.png)
(BTW, "true" and "false" should be translated, but I have noticed there is another thread regarding that.)
